src/journal/journald-server.c

```diff
@@ -2214,6 +2214,30 @@ static void vl_disconnect(VarlinkServer *server, Varlink *link, void *userdata)
         (void) server_start_or_stop_idle_timer(s); /* maybe we are idle now */
 }
 
+static int vl_subscribe_var(Varlink *link, JsonVariant *parameters, VarlinkMethodFlags flags, void *userdata) {
```

Comment:
Please drop `_var` from the function name.

Also, please move the function above `vl_connect()`.